Mathlib/Analysis/NormedSpace/FunctionSeries.lean

```diff
@@ -59,6 +99,24 @@ theorem tendstoUniformly_tsum_nat {f : ℕ → β → F} {u : ℕ → ℝ} (hu :
       atTop :=
   fun v hv => tendsto_finset_range.eventually (tendstoUniformly_tsum hu hfu v hv)
 
+/-- An infinite sum of functions with eventually summable sup norm is the uniform limit of its
+partial sums. Version with general index set. -/
+theorem tendstoUniformly_tsum_eventually {ι : Type*} {f : ι → β → F} {u : ι → ℝ}
+    (hu : Summable u) (hfu : ∃ a, ∀ (b : Finset ι), a ⊆ b → ∀ x n, n ∉ b → ‖f n x‖ ≤ u n) :
+    TendstoUniformly (fun t x => ∑ n ∈ t, f n x) (fun x => ∑' n, f n x) atTop := by
+  rw [← tendstoUniformlyOn_univ]
+  apply tendstoUniformlyOn_tsum_eventually hu
+  simpa using hfu
+
+/-- An infinite sum of functions with eventually summable sup norm is the uniform limit of its
+partial sums. Version with index set `ℕ`. -/
+theorem tendstoUniformly_tsum_nat_eventually {f : ℕ → α → F} {u : ℕ → ℝ}
+    (hu : Summable u) (hfu : ∀ᶠ n in atTop, ∀ x, ‖f n x‖ ≤ u n) :
```

Comment:
I think by reformulating the previous ones, this is not so necessary anymore, as the user can just swap `cofinite ℕ` for `atTop` whenever they wish.